### PR TITLE
(Procedure) Lots of UI

### PR DESCRIPTION
**Pull request by y-lohse**
This PR includes :

- The topbar navigation — not super final
- The overview screen — pretty accurate
- The Documents screen — has all the individual components, but doesn't use the template yet
- Amounts and duration screens but with nothing in them

### CODE REVIEW

File: packages/cozy-procedures/src/components/Overview.jsx [hunk 1]
```diff
@@ -1,36 +1,90 @@
...
+  render() {
+    const { personalData, t } = this.props
 
-  return (
-    <div>
-      <h1>{creditApplicationTemplate.name}</h1>
-      <div>
-        <Link to={`${rootPath}other`}>Other</Link>
-      </div>
+    const personalDataFieldsTotal = Object.keys(personalData).length
```
Comment (cedricmessiant) [inline packages/cozy-procedures/src/components/Overview.jsx]: nit: I will probably move this in a separate function to be able to use it in the form.

File: packages/cozy-procedures/src/locales/en.json [hunk 2]
```diff
@@ -1,71 +1,95 @@
 {
-  "personalData": {
-    "title": "Personal data",
-    "subtitle": "Please complete the following informations to finalise your application"
+  "confirm": "Confirm",
+  "overview": {
+    "subtitle": "Please fill out the following information",
+    "request": "Requested loan",
+    "amount": "Amount",
+    "duration": "duration",
```
Comment (cedricmessiant) [inline packages/cozy-procedures/src/locales/en.json]: "Duration" ?

File: packages/cozy-procedures/src/locales/en.json [hunk 3]
```diff
@@ -1,71 +1,95 @@
...
+    "documents": "Documents",
+    "infos": "Personal information",
```
Comment (cedricmessiant) [inline packages/cozy-procedures/src/locales/en.json]: maybe use `personalData` as key here so it is consistent

File: packages/cozy-procedures/src/locales/en.json [hunk 4]
```diff
@@ -1,71 +1,95 @@
...
-    "units": {
-      "eurosNetPerMonth": "Net € per month",
-      "eurosPerMonth": "€ per month"
+  "amount": {
+    "title": "Amount"
+  },
+  "duration": {
+    "title": "Duration"
+  },
+  "personalDataForm": {
```
Comment (cedricmessiant) [inline packages/cozy-procedures/src/locales/en.json]: :+1:

File: packages/cozy-procedures/src/locales/fr.json [hunk 5]
```diff
@@ -1,71 +1,95 @@
...
-    "title": "Informations personnelles",
-    "subtitle": "Complétez les informations suivantes pour finaliser votre dossier"
+  "confirm": "Valider",
+  "overview": {
+    "subtitle": "Complétez les informations suivantes pour demander un crédit",
+    "request": "Crédit demandé",
+    "amount": "Montant",
+    "duration": "durée",
+    "documents": "Pièces justificatives",
+    "infos": "Informations personelles",
```
Comment (cedricmessiant) [inline packages/cozy-procedures/src/locales/fr.json]: "personnelles"

File: packages/cozy-procedures/src/locales/fr.json [hunk 6]
```diff
@@ -1,71 +1,95 @@
 {
-  "personalData": {
-    "title": "Informations personnelles",
-    "subtitle": "Complétez les informations suivantes pour finaliser votre dossier"
+  "confirm": "Valider",
+  "overview": {
+    "subtitle": "Complétez les informations suivantes pour demander un crédit",
+    "request": "Crédit demandé",
+    "amount": "Montant",
+    "duration": "durée",
```
Comment (cedricmessiant) [inline packages/cozy-procedures/src/locales/fr.json]: really lowercase ?

File: packages/cozy-procedures/src/locales/fr.json [hunk 7]
```diff
@@ -1,71 +1,95 @@
...
+    "documents": "Pièces justificatives",
+    "infos": "Informations personelles",
+    "notice": "En soumettant votre demande, la totalité des éléménts de ce dossier nous sera envoyé.",
```
Comment (cedricmessiant) [inline packages/cozy-procedures/src/locales/fr.json]: "élements"
"nous sera envoyée" (la totalité) ou "nous serons envoyés" (les éléments)

File: packages/cozy-procedures/src/locales/fr.json (hunk 6 above)
Comment (y-lohse) [reply packages/cozy-procedures/src/locales/fr.json]: Yeah, same with `duration`. In the UI it forms a sentence `Amount over duration`. Then again should the casing really be done in a translation file..?

**Pull request merged**